qemu/deps/windows_ga_install/get_package.py

```diff
@@ -0,0 +1,73 @@
+#!/bin/env python
+
+import sys
+import re
+
+import commands
+from argparse import ArgumentParser
+
+
+class GuestAgentPkg(object):
+
+    """
+    A guest agent package class
+    """
+
+    def __init__(self, build_tag, build_name, url):
+        self.build_tag = build_tag
+        self.build_name = build_name
+        self.server_url = url
+
+    def _run_shell_cmd(self, cmd):
+        (status, output) = commands.getstatusoutput(cmd)
+        if status:
+            raise Exception("the cmd %s didn't run successfully" % cmd)
+        return (status, output)
+
+    def get_latest_build(self):
+        cmd = 'brew latest-build %s %s' % (self.build_tag, self.build_name)
+        (status, output) = self._run_shell_cmd(cmd)
+        for line in output.splitlines():
+            if self.build_name in line:
+                return line.split()[0]
+        raise Exception("didn't get latest build name")
+
+    def get_build_url(self):
+        build_name = self.get_latest_build()
+        cmd = 'brew buildinfo %s | grep msi' % build_name
+        (status, output) = self._run_shell_cmd(cmd)
+        url_list = []
+        for package in output.splitlines():
+            url = re.sub(r'/mnt/redhat', self.server_url, package)
```

Comment:
it looks 
--topdir
--brewurl
--brewhub/--server
should be include in your args. more details please see /etc/brewkoji.conf